apis/low_level_debug/src/lib.rs

```diff
@@ -0,0 +1,73 @@
+#![no_std]
+
+use libtock_platform::{CommandReturn, Syscalls};
+
+/// The low-level debug API provides tools to diagnose userspace issues that make normal debugging workflows (e.g. printing to the console) difficult.
+///
+/// It allows libraries to print alert codes and apps to print numeric information using only the command system call.
+///
+/// # Example
+/// ```ignore
+/// use libtock_runtime::TockSyscalls;
+///
+/// // Uses the real syscall implementation.
+/// type LowLevelDebug = libtock_low_level_debug::LowLevelDebug<TockSyscalls>;
+///
+/// // Prints 0x45 and the app which called it.
+/// LowLevelDebug::print_1(0x45);
+/// ```
+pub struct LowLevelDebug<S: Syscalls>(S);
+
+impl<S: Syscalls> LowLevelDebug<S> {
+    /// Run a check against the low-level debug capsule to ensure it is working.
+    #[inline(always)]
+    pub fn driver_check() -> CommandReturn {
```

Comment:
`driver_check` should probably return a `bool`: `true` if the LowLevelDebug capsule is present, `false` otherwise.

Reply:
Done.